tests/pipelines/eks/awscli-cl2-load-with-addons.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: tekton.dev/v1beta1
 kind: Pipeline
 metadata:
-  name: awscli-eks-cl2loadtest-with-addons
+  name: sushima-awscli-eks-cl2loadtest-with-addons
```

Comment:
we need to remove `sushima` prefix

Reply:
I think this change is an older one. This has been removed